tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/CachingType1Parser.scala

```diff
@@ -1,44 +0,0 @@
-package at.forsyte.apalache.tla.typecheck
```

Comment:
Is this change related to the feature added in this PR?

Reply:
Yes. Caching is not consistent with type aliases. If we want to cache parsing results, we have to take the aliases into account. That seems to be too difficult. There is no good use case for caching anyways.